src/lib/components/DeckGLMap/layers/wells/wellsLayer.ts

```diff
@@ -19,6 +20,16 @@ export interface WellDataType {
     };
 }
 
+function handleClick(event) {
+    this.setLayerProps("wells-layer", { selectedFeature: event.object });
+    return true;
+}
+
+const defaultProps = {
+    onClick: handleClick,
```

Comment:
We probably need a mechanism to toggle the selection on and off, so we can disable it when drawing is active. Not necessarily in this PR.